zerver/lib/exceptions.py

```diff
@@ -519,6 +519,25 @@ def __init__(
         self.daily_limit_reached: bool = daily_limit_reached
 
 
+class DirectMessageInitiationError(JsonableError):  #  nocoverage
+    def __init__(self) -> None:
+        pass
+
+    @staticmethod
+    @override
+    def msg_format() -> str:
+        return _("You are not allowed to initiate direct message thread.")
+
+
+class DirectMessagePermissionError(JsonableError):
+    def __init__(self, is_nobody_group: bool) -> None:
+        if is_nobody_group:
+            msg = "Direct messages are disabled in this organization."
+        else:
+            msg = "You are not allowed to send direct messages to these recipient(s)."
+        super().__init__(msg)
```

Comment:
You should know by now that you cannot add any strings without translation tags. I'm fiddling with the branch, so can probably replace the strings while I'm here.

Reply:
The nocoverage used here is also wrong; we should just add test coverage if needed.

```diff
diff --git a/zerver/lib/exceptions.py b/zerver/lib/exceptions.py
index 8409f7c33f..672ef2f5cf 100644
--- a/zerver/lib/exceptions.py
+++ b/zerver/lib/exceptions.py
@@ -519,22 +519,22 @@ class InvitationError(JsonableError):
         self.daily_limit_reached: bool = daily_limit_reached
 
 
-class DirectMessageInitiationError(JsonableError):  #  nocoverage
+class DirectMessageInitiationError(JsonableError):
     def __init__(self) -> None:
         pass
 
     @staticmethod
     @override
     def msg_format() -> str:
-        return _("You are not allowed to initiate direct message thread.")
+        return _("You do not have permission to initiate direct message conversations.")
 
 
 class DirectMessagePermissionError(JsonableError):
     def __init__(self, is_nobody_group: bool) -> None:
         if is_nobody_group:
-            msg = "Direct messages are disabled in this organization."
+            msg = _("Direct messages are disabled in this organization.")
         else:
-            msg = "You are not allowed to send direct messages to these recipient(s)."
+            msg = _("You do not have permission to send direct messages to this recipient.")
         super().__init__(msg)
 
 
```